hieradata/node/auxtel-dc01.tu.lsst.org.yaml

```diff
@@ -0,0 +1,34 @@
+---
+profile::daq::daq_interface::hwaddr: "a0:36:9f:20:4a:76"
+profile::daq::daq_interface::uuid: "dced8e46-f782-472a-b6a8-b7876a22afff"
+profile::daq::daq_interface::was: "p2p2"
+profile::daq::daq_interface::mode: "dhcp-client"
```

Comment:
All of the `profile::daq::*` keys can be removed.  That was EL7 specific configuration.

Reply:
I will cleaned this up.